domains/berzcodetechnologies.json

```diff
@@ -0,0 +1,16 @@
+{
+    "owner": {
+        "username": "BerzCode",
+        "email": "saurabhkirve@gmail.com"
+    },
+    "record": {
+        "A": "["154.41.233.131"]",
+        "TXT": [
+            {
+                "name": "_github-pages-challenge-berzcode",
+                "value": "bafe74203bd3e05bb01b98f65606ab"
+            }
+        ]
```

Comment:
Your GitHub Pages verification TXT must be in a new file with the name GitHub provides you with a TXT as a string with the other value GitHub gives you.